utils/googleutils/googleutils.go

```diff
@@ -6,9 +6,21 @@ import (
 	"golang.org/x/oauth2/google"
 )
 
+const (
+	EMPTY_CREDS = "{}"
+)
+
 func CompatibleGoogleCredentialsJSON(jsonKey []byte) error {
```

Comment:
[Nitpicking]
```suggestion
func CheckCredentialsJSONCompatibility(jsonKey []byte) error {
```

Reply:
@aris1009 can you let us know if this name change makes more sense ?

Reply:
Why do you want to defer a single method name change? I'd defer it if it results in several changes in the code.

Reply:
This PR is related to changes being made for GKE or workload identity and refactoring is being done only on the logic involved with it but this seems more like clean-up  wherein we are not changing anything related to that method

Reply:
Ok. Raise another PR then.